ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/host1.md

```diff
@@ -5495,6 +5540,48 @@ interface Port-Channel131.10
 interface Port-Channel132
    profile test-interface-profile
    description Test_port-channel_interface-profile
+!
+interface Port-Channel133
+   description Test1_switchport_tap_tool
+   switchport tap native vlan 10
+   switchport tap identity 3
+   switchport tap mac-address dest 01:00:00:00:00:00 src 01:23:45:67:89:ab
+   switchport tap encapsulation gre protocol 0x2 feature header length 3 strip
+   switchport tap encapsulation gre protocol 0x3 feature header length 2 strip re-encapsulation ethernet
+   switchport tap encapsulation gre protocol 0x10 strip
+   switchport tap encapsulation gre strip
+   switchport tap encapsulation gre destination 1.1.1.1 source 1.1.1.2 protocol 0x0 strip
+   switchport tap encapsulation gre destination 1.1.1.1 source 1.1.1.2 strip
+   switchport tap encapsulation gre destination 1.1.1.3 source 1.1.1.4 strip
+   switchport tap encapsulation gre destination 2.1.1.2 protocol 0x1 strip
+   switchport tap encapsulation gre destination 2.1.1.2 protocol 0x2 feature header length 2 strip re-encapsulation ethernet
+   switchport tap mpls pop all
+   switchport tool mpls pop all
+   switchport tool encapsulation vn-tag strip
+   switchport tool encapsulation dot1br strip
+   switchport tap allowed vlan 25
+   switchport tool allowed vlan 23
+   switchport tool identity qinq
+   switchport tool identity qinq source dzgre port inner policy
+   switchport tap truncation
+   switchport tool truncation 160
+   switchport tap default group g1 group g2 group g3
+   switchport tap default nexthop-group nexthop_g1 nexthop_g2 nexthop_g3
+   switchport tap default interface ethernet4
+   switchport tap default interface port-channel10
+   switchport tool group set group1 group2 group3
+   switchport tool dot1q remove outer 1-2
+   switchport tool dzgre preserve
```

Comment:
```
site3-leaf1(config-s-42-if-Po133)#   switchport tool dzgre preserve
% Invalid input

Reply:
ok there is an internal bug - this command has been removed so lets not add it (I don't have it in 4.32 nor 4.33) same for port-channel - it is not intended to be supported as per the internal bug description

Reply:
Removed this command